cmd/ddev/cmd/delete-images.go

```diff
@@ -130,3 +122,21 @@ func init() {
 	DeleteImagesCmd.Flags().BoolVarP(&deleteAllImages, "all", "a", false, "If set, deletes all Docker images created by ddev.")
 	DeleteCmd.AddCommand(DeleteImagesCmd)
 }
+
+// deleteDdevImages removes Docker images prefixed with ddev-
+func deleteDdevImages(images []docker.APIImages) error {
```

Comment:
As above, I think this can be done more completely by making this function find the images itself. Then the arg would be `deleteAll bool`. Without deleteAll it would just do old images.

Reply:
How do we define "old" images?

Reply:
The "old" images are the ones that don't go with the current version. That's what you did in the delete images work - looking to see what matched current and what didn't. And the --all was all of them regardless of match.